src/rdb.h

```diff
@@ -93,10 +93,11 @@
 #define RDB_TYPE_STREAM_LISTPACKS 15
 #define RDB_TYPE_HASH_LISTPACK 16
 #define RDB_TYPE_ZSET_LISTPACK 17
+#define RDB_TYPE_LIST_QUICKLIST_2   18
```

Comment:
Forget to add `quicklist-v2` to https://github.com/redis/redis/blob/cf860df59921efcc74be410bdf165abd784df502/src/redis-check-rdb.c#L95

Reply:
should I change  also quicklist to quicklist-v1 ?